rubicon/objc/runtime.py

```diff
@@ -811,10 +811,16 @@ def __call__(self, receiver, *args, convert_args=True, convert_result=True):
                     arg = arg.value
 
                 if issubclass(argtype, objc_block):
-                    if isinstance(arg, Block):
+                    if arg is None:
+                        # allow for 'nil' block args, which some objc methods accept
+                        arg = ns_from_py(arg)
+                    elif isinstance(arg, Block):
                         arg = arg.block
```

Comment:
Now that `Block`s have `_as_parameter_`, this `elif` branch should not be needed anymore - I think we can remove this and change the `else` below to `elif callable(arg)`.

Reply:
Ok, done.